src/org/thoughtcrime/securesms/video/recode/VideoRecoder.java

```diff
@@ -847,31 +862,35 @@ public static boolean prepareVideo(Context context, int chatId, DcMsg msg) {
       DcHelper.getContext(context).prepareMsg(chatId, msg);
 
       // calulate bytes
-      vei.estimatedBytes = VideoRecoder.calculateEstimatedSize((float) resultDurationMs / vei.originalDurationMs,
-          vei.resultVideoBitrate, vei.originalDurationMs, vei.originalAudioBytes);
+//      vei.estimatedBytes = VideoRecoder.calculateEstimatedSize((float) resultDurationMs / vei.originalDurationMs,
+//          vei.resultVideoBitrate, vei.originalDurationMs, vei.originalAudioBytes);
 
-      if (vei.estimatedBytes > MAX_BYTES+MAX_BYTES/4) {
-        logNtoast(context, String.format("recoding for %s failed: resulting file probably too large", inPath));
-        return false;
-      }
+//      if (vei.estimatedBytes > MAX_BYTES+MAX_BYTES/4) {
+//        logNtoast(context, String.format("recoding for %s failed: resulting file probably too large", inPath));
+//        return false;
+//      }
```

Comment:
why is that commented out?

iiuc, it is a rough check not even start recoding of a 1gig file.

however, vei.estimatedBytes could be a local variable - or removed if the check is really superfluous.

Reply:
This causes the video to be dropped.
So, if one makes a Video in DC about something and it was too large it was just gone permanently.
Currently it is present as a Draft and can be saved from there.
Alternative path here would be to store the raw version as a Draft instead and skip the recoding if its too large.